Get tests green: 7.1.x-next (2nd attempt)

ethanbustad opened the pull request.

Commit:
COMMERCE-700 Source formatting

Commit:
COMMERCE-0 Source formatting

Commit:
COMMERCE-677 Source formatting

Commit:
COMMERCE-721 Source formatting

Commit:
COMMERCE-721 Build language keys

Commit:
APIO-221 Deploy Apio config file to enable APIs

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

ethanbustad commented:
ci:test

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/com-liferay-commerce/tree/7.1.x-next">7.1.x-next</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-3.liferay.com/job/test-subrepository-acceptance-pullrequest(7.1.x)/25/">test-subrepository-acceptance-pullrequest(7.1.x)</a></p><p><b>For more details click <a href="https://test-1-3.liferay.com//userContent/jobs/test-subrepository-acceptance-pullrequest(7.1.x)/builds/25/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h1>Validation FAILED.</h1><p>Build Time: 12 minutes 54 seconds 774 ms</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/com-liferay-commerce/tree/7.1.x">7.1.x</a><br/>Branch GIT ID: <a href="https://github.com/liferay/com-liferay-commerce/commit/8b3754310f61af848ab5f9e0650c30b2e1232226">8b3754310f61af848ab5f9e0650c30b2e1232226</a></p><h6>Task Summary:</h6><ul><li>subrepository-compile-jdk8 -  :heavy_check_mark:</li><li>subrepository-semantic-versioning-jdk8 -  :heavy_check_mark:</li><li>subrepository-pmd-jdk8 -  :heavy_check_mark:</li><li>subrepository-source-format-jdk8 -  :x:<pre><code>     [echo] java.lang.Exception: Found 2 formatting issues:
     [echo] 1: ./commerce-service/src/main/java/com/liferay/commerce/internal/subscription/CommerceSubscriptionEntryHelperImpl.java expected:&lt;...());
     [echo] 
     [echo] 		newOrder.set[PaymentStatus(
     [echo] 			CommerceOrderConstants.PAYMENT_STATUS_PENDING);
     [echo] 		newOrder.setOrderStatus(
     [echo] 			CommerceOrderConstants.ORDER_STATUS_SUBSCRIPTION]);
     [echo] 
     [echo] 		newOrder = _co...&gt; but was:&lt;...());
     [echo] 
     [echo] 		newOrder.set[OrderStatus(
     [echo] 			CommerceOrderConstants.ORDER_STATUS_SUBSCRIPTION);
     [echo] 		newOrder.setPaymentStatus(
     [echo] 			CommerceOrderConstants.PAYMENT_STATUS_PENDING]);
     [echo] 
     [echo] 		newOrder = _co...&gt;
     [echo] 2: ./commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceSubscriptionEntryLocalServiceImpl.java expected:&lt;...ubscriptionEntry.set[NextIterationDate(nextInterationDate);
     [echo] 		commerceSubscriptionEntry.setStartDate(start]Date);
     [echo] 
     [echo] 		commerceSu...&gt; but was:&lt;...ubscriptionEntry.set[StartDate(startDate);
     [echo] 		commerceSubscriptionEntry.setNextIterationDate(nextInteration]Date);
     [echo] 
     [echo] 		commerceSu...&gt;
     [echo] 
     [echo] 	at com.liferay.source.formatter.SourceFormatter.format(SourceFormatter.java:412)
     [echo] 	at com.liferay.source.formatter.SourceFormatter.main(SourceFormatter.java:239)
     [echo] 
     [echo] FAILURE: Build failed with an exception.
     [echo] 
     [echo] * What went wrong:
     [echo] Execution failed for task ':checkSourceFormatting'.
     [echo] &gt; Process 'command '/opt/java/jdk1.8.0_121-x64/bin/java'' finished with non-zero exit value 1
     [echo] 
     [echo] * Try:
     [echo] Run with --info or --debug option to get more log output. Run with --scan to get full insights.
     [echo] 
     [echo] * Exception is:
     [echo] org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':checkSourceFormatting'.
     [echo] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:110)
     [echo] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:77)
     [echo] 	at org.gradle.api.internal.tasks.execution.OutputDirectoryCreatingTaskExecuter.execute(OutputDirectoryCreatingTaskExecuter.java:51)</code></pre></li><li>central-requirements-jdk8/subrepository-unit-jdk8 -  :heavy_check_mark:</li></ul><h5>For full console, click <a href="https://test-1-19.liferay.com/job/test-subrepository-acceptance-pullrequest-validation(7.1.x)/27//consoleText">here</a>.</h5><hr/><div><h6>Test Results:</h6><div><div><p>10 Tests Passed.<br/>0 Tests Failed.</p></div></div><h5>For all test results, click <a href="https://test-1-19.liferay.com/job/test-subrepository-acceptance-pullrequest-validation(7.1.x)/27//testReport">here</a>.</h5></div></html>

Commit:
COMMERCE-697 Source formatting

ethanbustad commented:
ci:test

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/com-liferay-commerce/tree/7.1.x-next">7.1.x-next</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-11.liferay.com/job/test-subrepository-acceptance-pullrequest(7.1.x)/40/">test-subrepository-acceptance-pullrequest(7.1.x)</a></p><p><b>For more details click <a href="https://test-1-11.liferay.com//userContent/jobs/test-subrepository-acceptance-pullrequest(7.1.x)/builds/40/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h1>Validation PASSED. Running batch tests.</h1><p>Build Time: 33 minutes 26 seconds 736 ms</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/com-liferay-commerce/tree/7.1.x">7.1.x</a><br/>Branch GIT ID: <a href="https://github.com/liferay/com-liferay-commerce/commit/8b3754310f61af848ab5f9e0650c30b2e1232226">8b3754310f61af848ab5f9e0650c30b2e1232226</a></p><h6>Task Summary:</h6><ul><li>subrepository-compile-jdk8 -  :heavy_check_mark:</li><li>subrepository-semantic-versioning-jdk8 -  :heavy_check_mark:</li><li>subrepository-pmd-jdk8 -  :heavy_check_mark:</li><li>subrepository-source-format-jdk8 -  :heavy_check_mark:</li><li>central-requirements-jdk8/subrepository-unit-jdk8 -  :heavy_check_mark:</li></ul><h5>For full console, click <a href="https://test-1-18.liferay.com/job/test-subrepository-acceptance-pullrequest-validation(7.1.x)/9//consoleText">here</a>.</h5><hr/><div><h6>Test Results:</h6><div><div><p>10 Tests Passed.<br/>0 Tests Failed.</p></div></div><h5>For all test results, click <a href="https://test-1-18.liferay.com/job/test-subrepository-acceptance-pullrequest-validation(7.1.x)/9//testReport">here</a>.</h5></div></html>

liferay-continuous-integration commented:
<html><h3>:x: ci:test - 2 out of 17 jobs passed in 3 hours 4 minutes 21 seconds 690 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/com-liferay-commerce/tree/7.1.x">7.1.x</a><br/>Branch GIT ID: <a href="https://github.com/liferay/com-liferay-commerce/commit/8b3754310f61af848ab5f9e0650c30b2e1232226">8b3754310f61af848ab5f9e0650c30b2e1232226</a></p>2 out of 17 jobs PASSED<div><h4>15 Failed Jobs:</h4><ul><li><a href="https://test-1-11.liferay.com/job/test-subrepository-acceptance-pullrequest(7.1.x)/40/"><strike><strong>test-subrepository-acceptance-pullrequest(7.1.x)</strong></strike></a></li><li><a href="https://test-1-7.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/65/"><strike><strong>test-subrepository-acceptance-pullrequest-batch(7.1.x)/functional-smoke-tomcat90-mysql57-jdk8/0</strong></strike></a></li><li><a href="https://test-1-4.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/62/"><strike><strong>test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-assemble-jdk8</strong></strike></a></li><li><a href="https://test-1-16.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/118/"><strike><strong>test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-functional-tomcat90-mysql57-jdk8/0</strong></strike></a></li><li><a href="https://test-1-6.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/120/"><strike><strong>test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-functional-tomcat90-mysql57-jdk8/1</strong></strike></a></li><li><a href="https://test-1-20.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/76/"><strike><strong>test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-functional-tomcat90-mysql57-jdk8/10</strong></strike></a></li><li><a href="https://test-1-16.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/119/"><strike><strong>test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-functional-tomcat90-mysql57-jdk8/2</strong></strike></a></li><li><a href="https://test-1-20.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/75/"><strike><strong>test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-functional-tomcat90-mysql57-jdk8/3</strong></strike></a></li><li><a href="https://test-1-23.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/182/"><strike><strong>test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-functional-tomcat90-mysql57-jdk8/4</strong></strike></a></li><li><a href="https://test-1-6.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/121/"><strike><strong>test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-functional-tomcat90-mysql57-jdk8/5</strong></strike></a></li><li><a href="https://test-1-14.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/109/"><strike><strong>test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-functional-tomcat90-mysql57-jdk8/6</strong></strike></a></li><li><a href="https://test-1-6.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/122/"><strike><strong>test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-functional-tomcat90-mysql57-jdk8/7</strong></strike></a></li><li><a href="https://test-1-14.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/110/"><strike><strong>test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-functional-tomcat90-mysql57-jdk8/8</strong></strike></a></li><li><a href="https://test-1-23.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/183/"><strike><strong>test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-functional-tomcat90-mysql57-jdk8/9</strong></strike></a></li><li><a href="https://test-1-19.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/136/"><strike><strong>test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-integration-mysql57-jdk8</strong></strike></a></li></ul></div><details><summary><strong>2 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-3.liferay.com/job/test-subrepository-acceptance-pullrequest-dist(7.1.x)/23/">test-subrepository-acceptance-pullrequest-dist(7.1.x)/jdk8</a></li><li><a href="https://test-1-18.liferay.com/job/test-subrepository-acceptance-pullrequest-validation(7.1.x)/9/">test-subrepository-acceptance-pullrequest-validation(7.1.x)/jdk8</a></li></ul></details><h5>For more details click <a href="https://test-1-11.liferay.com/userContent/jobs/test-subrepository-acceptance-pullrequest(7.1.x)/builds/40/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-7.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/65/">test-subrepository-acceptance-pullrequest-batch(7.1.x)/functional-smoke-tomcat90-mysql57-jdk8/0</a></h5><div><h6>Job Results:</h6><p>0 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-7.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/AXIS_VARIABLE=0,label_exp=!master/65//consoleText">AXIS_VARIABLE=0,label_exp=!master #65</a><div><p>Startup error: <strong>Unresolved Requirement(s)</strong></p><pre><code>     [exec] org.osgi.framework.BundleException: Could not resolve module: com.liferay.commerce.shipping.origin.locator.warehouse [478]
     [exec]   Unresolved requirement: Import-Package: com.liferay.commerce.model; version="[1.1.0,1.2.0)"
     [exec] 
     [exec] 	at org.eclipse.osgi.container.Module.start(Module.java:444)
     [exec] 	at org.eclipse.osgi.internal.framework.EquinoxBundle.start(EquinoxBundle.java:428)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startBundle(DirectoryWatcher.java:1258)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startBundles(DirectoryWatcher.java:1230)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startAllBundles(DirectoryWatcher.java:1218)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.doProcess(DirectoryWatcher.java:507)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.process(DirectoryWatcher.java:361)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.run(DirectoryWatcher.java:312)
     [exec] 04:10:28,045 ERROR [fileinstall-/opt/dev/projects/github/liferay-portal-7.1.x/bundles/osgi/modules][org_apache_felix_fileinstall:112] Exiting the JVM 
     [exec] org.osgi.framework.BundleException: Could not resolve module: com.liferay.commerce.shipping.origin.locator.warehouse [478]
     [exec]   Unresolved requirement: Import-Package: com.liferay.commerce.model; version="[1.1.0,1.2.0)"
     [exec] 
     [exec] 	at org.eclipse.osgi.container.Module.start(Module.java:444)
     [exec] 	at org.eclipse.osgi.internal.framework.EquinoxBundle.start(EquinoxBundle.java:428)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startBundle(DirectoryWatcher.java:1258)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startBundles(DirectoryWatcher.java:1230)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startAllBundles(DirectoryWatcher.java:1218)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.doProcess(DirectoryWatcher.java:507)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.process(DirectoryWatcher.java:361)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.run(DirectoryWatcher.java:312)</code></pre></div></div></li></ol></div></li><li><div><h5><a href="https://test-1-4.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/62/">test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-assemble-jdk8</a></h5><div><h6>Job Results:</h6><p>0 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-4.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/AXIS_VARIABLE=0,label_exp=!master/62//consoleText">AXIS_VARIABLE=0,label_exp=!master #62</a><pre><code>     [echo] 	at org.gradle.api.internal.project.taskfactory.StandardTaskAction.doExecute(StandardTaskAction.java:46)
     [echo] 	at org.gradle.api.internal.project.taskfactory.StandardTaskAction.execute(StandardTaskAction.java:39)
     [echo] 	at org.gradle.api.internal.project.taskfactory.StandardTaskAction.execute(StandardTaskAction.java:26)
     [echo] 	at org.gradle.api.internal.AbstractTask$TaskActionWrapper.execute(AbstractTask.java:801)
     [echo] 	at org.gradle.api.internal.AbstractTask$TaskActionWrapper.execute(AbstractTask.java:768)
     [echo] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter$1.run(ExecuteActionsTaskExecuter.java:131)
     [echo] 	at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:300)
     [echo] 	at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:292)
     [echo] 	at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:174)
     [echo] 	at org.gradle.internal.operations.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:90)
     [echo] 	at org.gradle.internal.operations.DelegatingBuildOperationExecutor.run(DelegatingBuildOperationExecutor.java:31)
     [echo] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeAction(ExecuteActionsTaskExecuter.java:120)
     [echo] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:99)
     [echo] 	... 31 more
     [echo] 
     [echo] 
     [echo] * Get more help at https://help.gradle.org
     [echo] 
     [echo] BUILD FAILED in 5m 34s
     [echo] The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal-7.1.x/build-common.xml:847: /opt/dev/projects/github/liferay-portal-7.1.x/build-common.xml:849: exec returned: 1
      [get] Getting: http://test-1-4/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/AXIS_VARIABLE=0,label_exp=!master/62//consoleText
      [get] To: /opt/dev/projects/github/liferay-portal-7.1.x/20181204035549005.txt
   [delete] Deleting: /opt/dev/projects/github/liferay-portal-7.1.x/20181204035549005.txt
   [delete] Deleting: /opt/dev/projects/github/liferay-portal-7.1.x/null1928318651.properties</code></pre></div></li></ol></div></li><li><div><h5><a href="https://test-1-16.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/118/">test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-functional-tomcat90-mysql57-jdk8/0</a></h5><div><h6>Job Results:</h6><p>0 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-16.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/AXIS_VARIABLE=0,label_exp=!master/118//consoleText">AXIS_VARIABLE=0,label_exp=!master #118</a><div><p>Startup error: <strong>Unresolved Requirement(s)</strong></p><pre><code>     [exec] org.osgi.framework.BundleException: Could not resolve module: com.liferay.commerce.shipping.origin.locator.warehouse [478]
     [exec]   Unresolved requirement: Import-Package: com.liferay.commerce.model; version="[1.1.0,1.2.0)"
     [exec] 
     [exec] 	at org.eclipse.osgi.container.Module.start(Module.java:444)
     [exec] 	at org.eclipse.osgi.internal.framework.EquinoxBundle.start(EquinoxBundle.java:428)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startBundle(DirectoryWatcher.java:1258)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startBundles(DirectoryWatcher.java:1230)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startAllBundles(DirectoryWatcher.java:1218)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.doProcess(DirectoryWatcher.java:507)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.process(DirectoryWatcher.java:361)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.run(DirectoryWatcher.java:312)
     [exec] 03:56:57,476 ERROR [fileinstall-/opt/dev/projects/github/liferay-portal-7.1.x/bundles/osgi/modules][org_apache_felix_fileinstall:112] Exiting the JVM 
     [exec] org.osgi.framework.BundleException: Could not resolve module: com.liferay.commerce.shipping.origin.locator.warehouse [478]
     [exec]   Unresolved requirement: Import-Package: com.liferay.commerce.model; version="[1.1.0,1.2.0)"
     [exec] 
     [exec] 	at org.eclipse.osgi.container.Module.start(Module.java:444)
     [exec] 	at org.eclipse.osgi.internal.framework.EquinoxBundle.start(EquinoxBundle.java:428)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startBundle(DirectoryWatcher.java:1258)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startBundles(DirectoryWatcher.java:1230)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startAllBundles(DirectoryWatcher.java:1218)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.doProcess(DirectoryWatcher.java:507)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.process(DirectoryWatcher.java:361)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.run(DirectoryWatcher.java:312)</code></pre></div></div></li></ol></div></li><li><div><h5><a href="https://test-1-6.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/120/">test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-functional-tomcat90-mysql57-jdk8/1</a></h5><div><h6>Job Results:</h6><p>0 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-6.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/AXIS_VARIABLE=0,label_exp=!master/120//consoleText">AXIS_VARIABLE=0,label_exp=!master #120</a><div><p>Startup error: <strong>Unresolved Requirement(s)</strong></p><pre><code>     [exec] org.osgi.framework.BundleException: Could not resolve module: com.liferay.commerce.shipping.origin.locator.warehouse [478]
     [exec]   Unresolved requirement: Import-Package: com.liferay.commerce.model; version="[1.1.0,1.2.0)"
     [exec] 
     [exec] 	at org.eclipse.osgi.container.Module.start(Module.java:444)
     [exec] 	at org.eclipse.osgi.internal.framework.EquinoxBundle.start(EquinoxBundle.java:428)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startBundle(DirectoryWatcher.java:1258)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startBundles(DirectoryWatcher.java:1230)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startAllBundles(DirectoryWatcher.java:1218)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.doProcess(DirectoryWatcher.java:507)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.process(DirectoryWatcher.java:361)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.run(DirectoryWatcher.java:312)
     [exec] 03:51:25,474 ERROR [fileinstall-/opt/dev/projects/github/liferay-portal-7.1.x/bundles/osgi/modules][org_apache_felix_fileinstall:112] Exiting the JVM 
     [exec] org.osgi.framework.BundleException: Could not resolve module: com.liferay.commerce.shipping.origin.locator.warehouse [478]
     [exec]   Unresolved requirement: Import-Package: com.liferay.commerce.model; version="[1.1.0,1.2.0)"
     [exec] 
     [exec] 	at org.eclipse.osgi.container.Module.start(Module.java:444)
     [exec] 	at org.eclipse.osgi.internal.framework.EquinoxBundle.start(EquinoxBundle.java:428)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startBundle(DirectoryWatcher.java:1258)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startBundles(DirectoryWatcher.java:1230)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startAllBundles(DirectoryWatcher.java:1218)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.doProcess(DirectoryWatcher.java:507)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.process(DirectoryWatcher.java:361)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.run(DirectoryWatcher.java:312)</code></pre></div></div></li></ol></div></li><li><div><h5><a href="https://test-1-20.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/76/">test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-functional-tomcat90-mysql57-jdk8/10</a></h5><div><h6>Job Results:</h6><p>0 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-20.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/AXIS_VARIABLE=0,label_exp=!master/76//consoleText">AXIS_VARIABLE=0,label_exp=!master #76</a><div><p>Startup error: <strong>Unresolved Requirement(s)</strong></p><pre><code>     [exec] org.osgi.framework.BundleException: Could not resolve module: com.liferay.commerce.shipping.origin.locator.warehouse [478]
     [exec]   Unresolved requirement: Import-Package: com.liferay.commerce.model; version="[1.1.0,1.2.0)"
     [exec] 
     [exec] 	at org.eclipse.osgi.container.Module.start(Module.java:444)
     [exec] 	at org.eclipse.osgi.internal.framework.EquinoxBundle.start(EquinoxBundle.java:428)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startBundle(DirectoryWatcher.java:1258)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startBundles(DirectoryWatcher.java:1230)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startAllBundles(DirectoryWatcher.java:1218)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.doProcess(DirectoryWatcher.java:507)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.process(DirectoryWatcher.java:361)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.run(DirectoryWatcher.java:312)
     [exec] 03:51:15,355 ERROR [fileinstall-/opt/dev/projects/github/liferay-portal-7.1.x/bundles/osgi/modules][org_apache_felix_fileinstall:112] Exiting the JVM 
     [exec] org.osgi.framework.BundleException: Could not resolve module: com.liferay.commerce.shipping.origin.locator.warehouse [478]
     [exec]   Unresolved requirement: Import-Package: com.liferay.commerce.model; version="[1.1.0,1.2.0)"
     [exec] 
     [exec] 	at org.eclipse.osgi.container.Module.start(Module.java:444)
     [exec] 	at org.eclipse.osgi.internal.framework.EquinoxBundle.start(EquinoxBundle.java:428)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startBundle(DirectoryWatcher.java:1258)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startBundles(DirectoryWatcher.java:1230)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startAllBundles(DirectoryWatcher.java:1218)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.doProcess(DirectoryWatcher.java:507)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.process(DirectoryWatcher.java:361)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.run(DirectoryWatcher.java:312)</code></pre></div></div></li></ol></div></li><li>...</li></ol></details></html>

Commit:
COMMERCE-0 Missed a few commerce-api project dependencies

ethanbustad commented:
ci:test

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/com-liferay-commerce/tree/7.1.x-next">7.1.x-next</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-6.liferay.com/job/test-subrepository-acceptance-pullrequest(7.1.x)/43/">test-subrepository-acceptance-pullrequest(7.1.x)</a></p><p><b>For more details click <a href="https://test-1-6.liferay.com//userContent/jobs/test-subrepository-acceptance-pullrequest(7.1.x)/builds/43/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h1>Validation PASSED. Running batch tests.</h1><p>Build Time: 14 minutes 2 seconds 84 ms</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/com-liferay-commerce/tree/7.1.x">7.1.x</a><br/>Branch GIT ID: <a href="https://github.com/liferay/com-liferay-commerce/commit/8b3754310f61af848ab5f9e0650c30b2e1232226">8b3754310f61af848ab5f9e0650c30b2e1232226</a></p><h6>Task Summary:</h6><ul><li>subrepository-compile-jdk8 -  :heavy_check_mark:</li><li>subrepository-semantic-versioning-jdk8 -  :heavy_check_mark:</li><li>subrepository-pmd-jdk8 -  :heavy_check_mark:</li><li>subrepository-source-format-jdk8 -  :heavy_check_mark:</li><li>central-requirements-jdk8/subrepository-unit-jdk8 -  :heavy_check_mark:</li></ul><h5>For full console, click <a href="https://test-1-2.liferay.com/job/test-subrepository-acceptance-pullrequest-validation(7.1.x)/22//consoleText">here</a>.</h5><hr/><div><h6>Test Results:</h6><div><div><p>10 Tests Passed.<br/>0 Tests Failed.</p></div></div><h5>For all test results, click <a href="https://test-1-2.liferay.com/job/test-subrepository-acceptance-pullrequest-validation(7.1.x)/22//testReport">here</a>.</h5></div></html>

Commit:
COMMERCE-628 Make sure Country fields are unique

We have unique indexes on `numericISOCode` and `twoLettersISOCode`
that we need to be sure not to violate when making multiple
Countries in this test class.

ethanbustad commented:
ci:stop

liferay-continuous-integration commented:
<h1>Unable to generate test report.</h1><p>Jenkins encountered the following error while generating test report:</p><pre><code>Sourced file: inline evaluation of: `` 						 							import com.liferay.jenkins.results.parser.Build; 							import co . . . '' : TargetError : at Line: 43 : in file: inline evaluation of: `` 						 							import com.liferay.jenkins.results.parser.Build; 							import co . . . '' : throw e ; 

Target exception: java.lang.NullPointerException
</code></pre><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/com-liferay-commerce/tree/7.1.x-next">7.1.x-next</a></p><p>Branch GIT ID: <a href="https://github.com/liferay/com-liferay-commerce/commit/8b3754310f61af848ab5f9e0650c30b2e1232226">8b3754310f61af848ab5f9e0650c30b2e1232226</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-6.liferay.com/job/test-subrepository-acceptance-pullrequest(7.1.x)/43/">test-subrepository-acceptance-pullrequest(7.1.x)</a></p><p><b>For more details click <a href="https://test-1-6.liferay.com//userContent/jobs/test-subrepository-acceptance-pullrequest(7.1.x)/builds/43/jenkins-report.html">here</a>.</b></p>

ethanbustad commented:
ci:test

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/com-liferay-commerce/tree/7.1.x-next">7.1.x-next</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-11.liferay.com/job/test-subrepository-acceptance-pullrequest(7.1.x)/41/">test-subrepository-acceptance-pullrequest(7.1.x)</a></p><p><b>For more details click <a href="https://test-1-11.liferay.com//userContent/jobs/test-subrepository-acceptance-pullrequest(7.1.x)/builds/41/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h1>Validation PASSED. Running batch tests.</h1><p>Build Time: 13 minutes 18 seconds 989 ms</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/com-liferay-commerce/tree/7.1.x">7.1.x</a><br/>Branch GIT ID: <a href="https://github.com/liferay/com-liferay-commerce/commit/8b3754310f61af848ab5f9e0650c30b2e1232226">8b3754310f61af848ab5f9e0650c30b2e1232226</a></p><h6>Task Summary:</h6><ul><li>subrepository-compile-jdk8 -  :heavy_check_mark:</li><li>subrepository-semantic-versioning-jdk8 -  :heavy_check_mark:</li><li>subrepository-pmd-jdk8 -  :heavy_check_mark:</li><li>subrepository-source-format-jdk8 -  :heavy_check_mark:</li><li>central-requirements-jdk8/subrepository-unit-jdk8 -  :heavy_check_mark:</li></ul><h5>For full console, click <a href="https://test-1-21.liferay.com/job/test-subrepository-acceptance-pullrequest-validation(7.1.x)/32//consoleText">here</a>.</h5><hr/><div><h6>Test Results:</h6><div><div><p>10 Tests Passed.<br/>0 Tests Failed.</p></div></div><h5>For all test results, click <a href="https://test-1-21.liferay.com/job/test-subrepository-acceptance-pullrequest-validation(7.1.x)/32//testReport">here</a>.</h5></div></html>

liferay-continuous-integration commented:
<html><h3>:x: ci:test - 2 out of 17 jobs passed in 1 hour 16 minutes 12 seconds 988 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/com-liferay-commerce/tree/7.1.x">7.1.x</a><br/>Branch GIT ID: <a href="https://github.com/liferay/com-liferay-commerce/commit/8b3754310f61af848ab5f9e0650c30b2e1232226">8b3754310f61af848ab5f9e0650c30b2e1232226</a></p>2 out of 17 jobs PASSED<div><h4>15 Failed Jobs:</h4><ul><li><a href="https://test-1-11.liferay.com/job/test-subrepository-acceptance-pullrequest(7.1.x)/41/"><strike><strong>test-subrepository-acceptance-pullrequest(7.1.x)</strong></strike></a></li><li><a href="https://test-1-7.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/67/"><strike><strong>test-subrepository-acceptance-pullrequest-batch(7.1.x)/functional-smoke-tomcat90-mysql57-jdk8/0</strong></strike></a></li><li><a href="https://test-1-5.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/129/"><strike><strong>test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-assemble-jdk8</strong></strike></a></li><li><a href="https://test-1-3.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/126/"><strike><strong>test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-functional-tomcat90-mysql57-jdk8/0</strong></strike></a></li><li><a href="https://test-1-7.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/66/"><strike><strong>test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-functional-tomcat90-mysql57-jdk8/1</strong></strike></a></li><li><a href="https://test-1-14.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/117/"><strike><strong>test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-functional-tomcat90-mysql57-jdk8/10</strong></strike></a></li><li><a href="https://test-1-13.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/74/"><strike><strong>test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-functional-tomcat90-mysql57-jdk8/2</strong></strike></a></li><li><a href="https://test-1-18.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/87/"><strike><strong>test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-functional-tomcat90-mysql57-jdk8/3</strong></strike></a></li><li><a href="https://test-1-19.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/143/"><strike><strong>test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-functional-tomcat90-mysql57-jdk8/4</strong></strike></a></li><li><a href="https://test-1-5.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/128/"><strike><strong>test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-functional-tomcat90-mysql57-jdk8/5</strong></strike></a></li><li><a href="https://test-1-3.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/127/"><strike><strong>test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-functional-tomcat90-mysql57-jdk8/6</strong></strike></a></li><li><a href="https://test-1-4.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/66/"><strike><strong>test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-functional-tomcat90-mysql57-jdk8/7</strong></strike></a></li><li><a href="https://test-1-21.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/92/"><strike><strong>test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-functional-tomcat90-mysql57-jdk8/8</strong></strike></a></li><li><a href="https://test-1-9.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/32/"><strike><strong>test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-functional-tomcat90-mysql57-jdk8/9</strong></strike></a></li><li><a href="https://test-1-13.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/75/"><strike><strong>test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-integration-mysql57-jdk8</strong></strike></a></li></ul></div><details><summary><strong>2 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-21.liferay.com/job/test-subrepository-acceptance-pullrequest-dist(7.1.x)/21/">test-subrepository-acceptance-pullrequest-dist(7.1.x)/jdk8</a></li><li><a href="https://test-1-21.liferay.com/job/test-subrepository-acceptance-pullrequest-validation(7.1.x)/32/">test-subrepository-acceptance-pullrequest-validation(7.1.x)/jdk8</a></li></ul></details><h5>For more details click <a href="https://test-1-11.liferay.com/userContent/jobs/test-subrepository-acceptance-pullrequest(7.1.x)/builds/41/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-7.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/67/">test-subrepository-acceptance-pullrequest-batch(7.1.x)/functional-smoke-tomcat90-mysql57-jdk8/0</a></h5><div><h6>Job Results:</h6><p>0 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-7.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/AXIS_VARIABLE=0,label_exp=!master/67//consoleText">AXIS_VARIABLE=0,label_exp=!master #67</a><div><p>Startup error: <strong>Unresolved Requirement(s)</strong></p><pre><code>     [exec] org.osgi.framework.BundleException: Could not resolve module: com.liferay.commerce.shipping.origin.locator.address [477]
     [exec]   Unresolved requirement: Import-Package: com.liferay.commerce.model; version="[1.1.0,1.2.0)"
     [exec] 
     [exec] 	at org.eclipse.osgi.container.Module.start(Module.java:444)
     [exec] 	at org.eclipse.osgi.internal.framework.EquinoxBundle.start(EquinoxBundle.java:428)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startBundle(DirectoryWatcher.java:1258)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startBundles(DirectoryWatcher.java:1230)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startAllBundles(DirectoryWatcher.java:1218)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.doProcess(DirectoryWatcher.java:507)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.process(DirectoryWatcher.java:361)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.run(DirectoryWatcher.java:312)
     [exec] 19:27:09,226 ERROR [fileinstall-/opt/dev/projects/github/liferay-portal-7.1.x/bundles/osgi/modules][org_apache_felix_fileinstall:112] Exiting the JVM 
     [exec] org.osgi.framework.BundleException: Could not resolve module: com.liferay.commerce.shipping.origin.locator.address [477]
     [exec]   Unresolved requirement: Import-Package: com.liferay.commerce.model; version="[1.1.0,1.2.0)"
     [exec] 
     [exec] 	at org.eclipse.osgi.container.Module.start(Module.java:444)
     [exec] 	at org.eclipse.osgi.internal.framework.EquinoxBundle.start(EquinoxBundle.java:428)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startBundle(DirectoryWatcher.java:1258)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startBundles(DirectoryWatcher.java:1230)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startAllBundles(DirectoryWatcher.java:1218)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.doProcess(DirectoryWatcher.java:507)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.process(DirectoryWatcher.java:361)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.run(DirectoryWatcher.java:312)</code></pre></div></div></li></ol></div></li><li><div><h5><a href="https://test-1-5.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/129/">test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-assemble-jdk8</a></h5><div><h6>Job Results:</h6><p>0 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-5.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/AXIS_VARIABLE=0,label_exp=!master/129//consoleText">AXIS_VARIABLE=0,label_exp=!master #129</a><pre><code>     [echo] 	at org.gradle.api.internal.project.taskfactory.StandardTaskAction.doExecute(StandardTaskAction.java:46)
     [echo] 	at org.gradle.api.internal.project.taskfactory.StandardTaskAction.execute(StandardTaskAction.java:39)
     [echo] 	at org.gradle.api.internal.project.taskfactory.StandardTaskAction.execute(StandardTaskAction.java:26)
     [echo] 	at org.gradle.api.internal.AbstractTask$TaskActionWrapper.execute(AbstractTask.java:801)
     [echo] 	at org.gradle.api.internal.AbstractTask$TaskActionWrapper.execute(AbstractTask.java:768)
     [echo] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter$1.run(ExecuteActionsTaskExecuter.java:131)
     [echo] 	at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:300)
     [echo] 	at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:292)
     [echo] 	at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:174)
     [echo] 	at org.gradle.internal.operations.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:90)
     [echo] 	at org.gradle.internal.operations.DelegatingBuildOperationExecutor.run(DelegatingBuildOperationExecutor.java:31)
     [echo] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeAction(ExecuteActionsTaskExecuter.java:120)
     [echo] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:99)
     [echo] 	... 31 more
     [echo] 
     [echo] 
     [echo] * Get more help at https://help.gradle.org
     [echo] 
     [echo] BUILD FAILED in 5m 46s
     [echo] The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal-7.1.x/build-common.xml:847: /opt/dev/projects/github/liferay-portal-7.1.x/build-common.xml:849: exec returned: 1
      [get] Getting: http://test-1-5/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/AXIS_VARIABLE=0,label_exp=!master/129//consoleText
      [get] To: /opt/dev/projects/github/liferay-portal-7.1.x/20181204193217441.txt
   [delete] Deleting: /opt/dev/projects/github/liferay-portal-7.1.x/20181204193217441.txt
   [delete] Deleting: /opt/dev/projects/github/liferay-portal-7.1.x/null309308000.properties</code></pre></div></li></ol></div></li><li><div><h5><a href="https://test-1-3.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/126/">test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-functional-tomcat90-mysql57-jdk8/0</a></h5><div><h6>Job Results:</h6><p>0 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-3.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/AXIS_VARIABLE=0,label_exp=!master/126//consoleText">AXIS_VARIABLE=0,label_exp=!master #126</a><div><p>Startup error: <strong>Unresolved Requirement(s)</strong></p><pre><code>     [exec] org.osgi.framework.BundleException: Could not resolve module: com.liferay.commerce.shipping.origin.locator.address [477]
     [exec]   Unresolved requirement: Import-Package: com.liferay.commerce.model; version="[1.1.0,1.2.0)"
     [exec] 
     [exec] 	at org.eclipse.osgi.container.Module.start(Module.java:444)
     [exec] 	at org.eclipse.osgi.internal.framework.EquinoxBundle.start(EquinoxBundle.java:428)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startBundle(DirectoryWatcher.java:1258)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startBundles(DirectoryWatcher.java:1230)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startAllBundles(DirectoryWatcher.java:1218)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.doProcess(DirectoryWatcher.java:507)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.process(DirectoryWatcher.java:361)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.run(DirectoryWatcher.java:312)
     [exec] 19:27:36,547 ERROR [fileinstall-/opt/dev/projects/github/liferay-portal-7.1.x/bundles/osgi/modules][org_apache_felix_fileinstall:112] Exiting the JVM 
     [exec] org.osgi.framework.BundleException: Could not resolve module: com.liferay.commerce.shipping.origin.locator.address [477]
     [exec]   Unresolved requirement: Import-Package: com.liferay.commerce.model; version="[1.1.0,1.2.0)"
     [exec] 
     [exec] 	at org.eclipse.osgi.container.Module.start(Module.java:444)
     [exec] 	at org.eclipse.osgi.internal.framework.EquinoxBundle.start(EquinoxBundle.java:428)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startBundle(DirectoryWatcher.java:1258)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startBundles(DirectoryWatcher.java:1230)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startAllBundles(DirectoryWatcher.java:1218)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.doProcess(DirectoryWatcher.java:507)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.process(DirectoryWatcher.java:361)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.run(DirectoryWatcher.java:312)</code></pre></div></div></li></ol></div></li><li><div><h5><a href="https://test-1-7.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/66/">test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-functional-tomcat90-mysql57-jdk8/1</a></h5><div><h6>Job Results:</h6><p>0 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-7.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/AXIS_VARIABLE=0,label_exp=!master/66//consoleText">AXIS_VARIABLE=0,label_exp=!master #66</a><div><p>Startup error: <strong>Unresolved Requirement(s)</strong></p><pre><code>     [exec] org.osgi.framework.BundleException: Could not resolve module: com.liferay.commerce.shipping.origin.locator.address [477]
     [exec]   Unresolved requirement: Import-Package: com.liferay.commerce.model; version="[1.1.0,1.2.0)"
     [exec] 
     [exec] 	at org.eclipse.osgi.container.Module.start(Module.java:444)
     [exec] 	at org.eclipse.osgi.internal.framework.EquinoxBundle.start(EquinoxBundle.java:428)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startBundle(DirectoryWatcher.java:1258)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startBundles(DirectoryWatcher.java:1230)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startAllBundles(DirectoryWatcher.java:1218)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.doProcess(DirectoryWatcher.java:507)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.process(DirectoryWatcher.java:361)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.run(DirectoryWatcher.java:312)
     [exec] 19:41:43,645 ERROR [fileinstall-/opt/dev/projects/github/liferay-portal-7.1.x/bundles/osgi/modules][org_apache_felix_fileinstall:112] Exiting the JVM 
     [exec] org.osgi.framework.BundleException: Could not resolve module: com.liferay.commerce.shipping.origin.locator.address [477]
     [exec]   Unresolved requirement: Import-Package: com.liferay.commerce.model; version="[1.1.0,1.2.0)"
     [exec] 
     [exec] 	at org.eclipse.osgi.container.Module.start(Module.java:444)
     [exec] 	at org.eclipse.osgi.internal.framework.EquinoxBundle.start(EquinoxBundle.java:428)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startBundle(DirectoryWatcher.java:1258)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startBundles(DirectoryWatcher.java:1230)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startAllBundles(DirectoryWatcher.java:1218)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.doProcess(DirectoryWatcher.java:507)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.process(DirectoryWatcher.java:361)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.run(DirectoryWatcher.java:312)</code></pre></div></div></li></ol></div></li><li><div><h5><a href="https://test-1-14.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/117/">test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-functional-tomcat90-mysql57-jdk8/10</a></h5><div><h6>Job Results:</h6><p>0 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-14.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/AXIS_VARIABLE=0,label_exp=!master/117//consoleText">AXIS_VARIABLE=0,label_exp=!master #117</a><div><p>Startup error: <strong>Unresolved Requirement(s)</strong></p><pre><code>     [exec] org.osgi.framework.BundleException: Could not resolve module: com.liferay.commerce.shipping.origin.locator.address [477]
     [exec]   Unresolved requirement: Import-Package: com.liferay.commerce.model; version="[1.1.0,1.2.0)"
     [exec] 
     [exec] 	at org.eclipse.osgi.container.Module.start(Module.java:444)
     [exec] 	at org.eclipse.osgi.internal.framework.EquinoxBundle.start(EquinoxBundle.java:428)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startBundle(DirectoryWatcher.java:1258)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startBundles(DirectoryWatcher.java:1230)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startAllBundles(DirectoryWatcher.java:1218)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.doProcess(DirectoryWatcher.java:507)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.process(DirectoryWatcher.java:361)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.run(DirectoryWatcher.java:312)
     [exec] 19:27:32,601 ERROR [fileinstall-/opt/dev/projects/github/liferay-portal-7.1.x/bundles/osgi/modules][org_apache_felix_fileinstall:112] Exiting the JVM 
     [exec] org.osgi.framework.BundleException: Could not resolve module: com.liferay.commerce.shipping.origin.locator.address [477]
     [exec]   Unresolved requirement: Import-Package: com.liferay.commerce.model; version="[1.1.0,1.2.0)"
     [exec] 
     [exec] 	at org.eclipse.osgi.container.Module.start(Module.java:444)
     [exec] 	at org.eclipse.osgi.internal.framework.EquinoxBundle.start(EquinoxBundle.java:428)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startBundle(DirectoryWatcher.java:1258)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startBundles(DirectoryWatcher.java:1230)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startAllBundles(DirectoryWatcher.java:1218)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.doProcess(DirectoryWatcher.java:507)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.process(DirectoryWatcher.java:361)
     [exec] 	at org.apache.felix.fileinstall.internal.DirectoryWatcher.run(DirectoryWatcher.java:312)</code></pre></div></div></li></ol></div></li><li>...</li></ol></details></html>

Commit:
COMMERCE-697 product.util has increased version

ethanbustad commented:
ci:test

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/com-liferay-commerce/tree/7.1.x-next">7.1.x-next</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-22.liferay.com/job/test-subrepository-acceptance-pullrequest(7.1.x)/61/">test-subrepository-acceptance-pullrequest(7.1.x)</a></p><p><b>For more details click <a href="https://test-1-22.liferay.com//userContent/jobs/test-subrepository-acceptance-pullrequest(7.1.x)/builds/61/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h1>Validation FAILED.</h1><p>Build Time: 20 minutes 15 seconds 682 ms</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/com-liferay-commerce/tree/7.1.x">7.1.x</a><br/>Branch GIT ID: <a href="https://github.com/liferay/com-liferay-commerce/commit/8b3754310f61af848ab5f9e0650c30b2e1232226">8b3754310f61af848ab5f9e0650c30b2e1232226</a></p><h6>Task Summary:</h6><ul><li>subrepository-compile-jdk8 -  :heavy_check_mark:</li><li>subrepository-semantic-versioning-jdk8 -  :heavy_check_mark:</li><li>subrepository-pmd-jdk8 -  :heavy_check_mark:</li><li>subrepository-source-format-jdk8 -  :x:<pre><code>     [echo] java.lang.Exception: Found 1 formatting issues:
     [echo] 1: ./commerce-data-integration/commerce-data-integration-apio-test/src/testIntegration/java/com/liferay/commerce/data/integration/apio/resource/test/CommerceCountryNestedCollectionResourceTest.java expected:&lt;...PropertiesMap;
     [echo] 	}
     [echo] 
     [echo] 	[@DeleteAfterTestRun
     [echo] 	private Group _group;
     [echo] 
     [echo] 	private final UniqueStringRandomizerBumper _randomizerBumper =
     [echo] 		new UniqueStringRandomizerBumper()];
     [echo] 
     [echo] }&gt; but was:&lt;...PropertiesMap;
     [echo] 	}
     [echo] 
     [echo] 	[private final UniqueStringRandomizerBumper _randomizerBumper =
     [echo] 		new UniqueStringRandomizerBumper();
     [echo] 
     [echo] 	@DeleteAfterTestRun
     [echo] 	private Group _group];
     [echo] 
     [echo] }&gt;
     [echo] 
     [echo] 	at com.liferay.source.formatter.SourceFormatter.format(SourceFormatter.java:412)
     [echo] 	at com.liferay.source.formatter.SourceFormatter.main(SourceFormatter.java:239)
     [echo] 
     [echo] FAILURE: Build failed with an exception.
     [echo] 
     [echo] * What went wrong:
     [echo] Execution failed for task ':checkSourceFormatting'.
     [echo] &gt; Process 'command '/opt/java/jdk1.8.0_121-x64/bin/java'' finished with non-zero exit value 1
     [echo] 
     [echo] * Try:
     [echo] Run with --info or --debug option to get more log output. Run with --scan to get full insights.
     [echo] 
     [echo] * Exception is:
     [echo] org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':checkSourceFormatting'.
     [echo] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:110)
     [echo] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:77)
     [echo] 	at org.gradle.api.internal.tasks.execution.OutputDirectoryCreatingTaskExecuter.execute(OutputDirectoryCreatingTaskExecuter.java:51)
     [echo] 	at org.gradle.api.internal.tasks.execution.SkipUpToDateTaskExecuter.execute(SkipUpToDateTaskExecuter.java:59)
     [echo] 	at org.gradle.api.internal.tasks.execution.ResolveTaskOutputCachingStateExecuter.execute(ResolveTaskOutputCachingStateExecuter.java:54)
     [echo] 	at org.gradle.api.internal.tasks.execution.ValidatingTaskExecuter.execute(ValidatingTaskExecuter.java:59)</code></pre></li><li>central-requirements-jdk8/subrepository-unit-jdk8 -  :heavy_check_mark:</li></ul><h5>For full console, click <a href="https://test-1-22.liferay.com/job/test-subrepository-acceptance-pullrequest-validation(7.1.x)/48//consoleText">here</a>.</h5><hr/><div><h6>Test Results:</h6><div><div><p>10 Tests Passed.<br/>0 Tests Failed.</p></div></div><h5>For all test results, click <a href="https://test-1-22.liferay.com/job/test-subrepository-acceptance-pullrequest-validation(7.1.x)/48//testReport">here</a>.</h5></div></html>

ethanbustad commented:
ci:test

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/com-liferay-commerce/tree/7.1.x-next">7.1.x-next</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-7.liferay.com/job/test-subrepository-acceptance-pullrequest(7.1.x)/15/">test-subrepository-acceptance-pullrequest(7.1.x)</a></p><p><b>For more details click <a href="https://test-1-7.liferay.com//userContent/jobs/test-subrepository-acceptance-pullrequest(7.1.x)/builds/15/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h1>Validation PASSED. Running batch tests.</h1><p>Build Time: 15 minutes 13 seconds 720 ms</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/com-liferay-commerce/tree/7.1.x">7.1.x</a><br/>Branch GIT ID: <a href="https://github.com/liferay/com-liferay-commerce/commit/8b3754310f61af848ab5f9e0650c30b2e1232226">8b3754310f61af848ab5f9e0650c30b2e1232226</a></p><h6>Task Summary:</h6><ul><li>subrepository-compile-jdk8 -  :heavy_check_mark:</li><li>subrepository-semantic-versioning-jdk8 -  :heavy_check_mark:</li><li>subrepository-pmd-jdk8 -  :heavy_check_mark:</li><li>subrepository-source-format-jdk8 -  :heavy_check_mark:</li><li>central-requirements-jdk8/subrepository-unit-jdk8 -  :heavy_check_mark:</li></ul><h5>For full console, click <a href="https://test-1-10.liferay.com/job/test-subrepository-acceptance-pullrequest-validation(7.1.x)/12//consoleText">here</a>.</h5><hr/><div><h6>Test Results:</h6><div><div><p>10 Tests Passed.<br/>0 Tests Failed.</p></div></div><h5>For all test results, click <a href="https://test-1-10.liferay.com/job/test-subrepository-acceptance-pullrequest-validation(7.1.x)/12//testReport">here</a>.</h5></div></html>

ethanbustad commented:
ci:stop

liferay-continuous-integration commented:
<h1>Unable to generate test report.</h1><p>Jenkins encountered the following error while generating test report:</p><pre><code>Sourced file: inline evaluation of: `` 						 							import com.liferay.jenkins.results.parser.Build; 							import co . . . '' : TargetError : at Line: 43 : in file: inline evaluation of: `` 						 							import com.liferay.jenkins.results.parser.Build; 							import co . . . '' : throw e ; 

Target exception: java.lang.NullPointerException
</code></pre><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/com-liferay-commerce/tree/7.1.x-next">7.1.x-next</a></p><p>Branch GIT ID: <a href="https://github.com/liferay/com-liferay-commerce/commit/8b3754310f61af848ab5f9e0650c30b2e1232226">8b3754310f61af848ab5f9e0650c30b2e1232226</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-7.liferay.com/job/test-subrepository-acceptance-pullrequest(7.1.x)/15/">test-subrepository-acceptance-pullrequest(7.1.x)</a></p><p><b>For more details click <a href="https://test-1-7.liferay.com//userContent/jobs/test-subrepository-acceptance-pullrequest(7.1.x)/builds/15/jenkins-report.html">here</a>.</b></p>

ethanbustad commented:
Resent to https://github.com/liferay/com-liferay-commerce/pull/285.
